tests/baselines/reference/api/typescript.d.ts

```diff
@@ -5846,6 +5846,14 @@ declare namespace ts {
         emit(writeFile?: WriteFileCallback, customTransformers?: CustomTransformers): EmitResult | BuildInvalidedProject<T> | undefined;
     }
     type InvalidatedProject<T extends BuilderProgram> = UpdateOutputFileStampsProject | BuildInvalidedProject<T> | UpdateBundleProject<T>;
+    namespace JsTyping {
```

Comment:
What do you mean? The TypingResolutionHost is something I really would have to implement, iiuc

Reply:
Sure, but this is typescript.d.ts, not tsserverlibrary.d.ts, and it's not going to contain the ATA stuff as far as I'm aware (note that the change to this file is type-only).

Reply:
I am pretty sure this is unnecessary if we just switch these to direct import instead of re-exporting from `src/services/_namespaces/ts.ts`.

Reply:
@DanielRosenwasser You mean doing `import { TypingResolutionHost } from "./jsTyping;"`? 'cause that broke the tests because it can't find it?? (sorry, I'm not entirely understanding how the namespace mapping thing works right now)

Reply:
Yeah, our bundling is unfortunately far too restrictive to allow this sort of thing right now. I think it's just going to have to be gross.

Reply:
Hold up, why can't the tests just do a direct import as well?

Reply:
Bump

Reply:
@zkat if you change existing instances of `TypingResolutionHost` to direct imports in the harness, does the bundler still break on that?

Reply:
Okay, I just realized that we (just below here) already have `ts.server` declared in `typescript.d.ts`. This isn't a regression; you can see the same thing in TS 4.9.5 too.

So, I think I'm okay with this PR as-is. It's no worse than our current API in terms of over-declaring a public API. ATA already leaks into `typescript.d.ts` as it is so I don't think there's really much to do here.